crowbar_framework/app/controllers/installer/upgrades_controller.rb

```diff
@@ -323,9 +323,14 @@ def post_cleanup_repos
           false
         )
 
-        Crowbar::Repository.chef_data_bag_destroy(
-          "#{repository.data_bag_name}/#{repository.id}"
-        )
+        begin
+          Crowbar::Repository.chef_data_bag_destroy(
+            "#{repository.data_bag_name}/#{repository.id}"
+          )
+        rescue Net::HTTPServerException
```

Comment:
Then no, it sounds fine. I did expect we could directly catch a HTTPServerException404 or something similar :disappointed: